core/modules/qproc/testQueryAnaGeneral.cc

```diff
@@ -1012,7 +1012,7 @@ BOOST_AUTO_TEST_CASE(Case01_1083) {
         "from Source s join RefObjMatch rom using (objectId) "
         "join SimRefObject sro using (refObjectId) where isStar =1 limit 10;";
     // % is not valid for arithmetic in SQL92
-    char const expectedErr[] = "ParseException:Error parsing query, near \"%\"";
+    char const expectedErr[] = "ParseException:Error parsing query, near \"%\", Unhandled operator type:%";
```

Comment:
I understand this code was there for year. However, it looks a bit dangerous to depend on a very specific error message.

Reply:
it's testing for an expected error message; if it changes I want to know.